importer/api/__init__.py

```diff
@@ -144,6 +142,16 @@ def import_children(course, element, parent):
         content_xml=etree.tostring(element),
         mpath=mpath,
     )
+    if element.tag == "video":
+        subname = get_video_sub(element)
+        if subname != "":
+            assets = StaticAsset.objects.filter(
+                course__id=resource.course_id,
+                path=subname,
+            )
+            for asset in assets:
+                resource.static_assets.add(asset)
```

Comment:
Nope, tried that already. Error. At least with `.add()`, and I don't see another method available in [the docs](https://docs.djangoproject.com/en/1.8/topics/db/examples/many_to_many/).